include/messages.h

```diff
@@ -42,7 +42,7 @@ extern "C"
 
 #define ENUM_TYPE uint8_t
 
-#define MAX_PACKED_STRING_LEN 1024
+#define MAX_PACKED_STRING_LEN (4 * 1024)
```

Comment:
Is it? The problem is with the extension and BiK lists which can be long.

Reply:
This define is used for statically sized arrays embedded in the command structs. And since the command structs hold a union of all possible commands, their memory footprint quickly balloons if some command contains a lot of strings or reserves a lot of space for worst-case strings. If the list can get too large it might be better to send it in extra_data or extra_data2 (depending on which one is still unused).